ChangeLog.d/issue4176.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix an issue where resource is never freed when running one particular
+     test suite with an alternative AES implementation. Fixes #4176.
```

Comment:
We're rather picky on good English in changelog entries.

* “Fix an issue where X happens” → ”Fix X“ — the extra words are just a distraction.
* “resource” → “a resource”
* “where resource is never freed” has a name: it's a resource leak
* “when running one particular test suite” → “in a test suite”

```suggestion
   * Fix a resource leak in a test suite with an alternative AES
     implementation. Fixes #4176.
```